Assets/Scripts/Enemies/Ratboss/RatbossA0.cs

```diff
@@ -4,10 +4,11 @@
 using System.Collections.Generic;
 using UnityEngine;
 
-public class RatbossA0 : MonoBehaviour
-{ 
+public class RatbossA0 : BossAction
```

Comment:
Okay so it does get inherited instead of MonoBehaviour
Please use `abstract` in the class definition to make that more clear

Reply:
will do